source/connect/ssh-connection.txt

```diff
@@ -11,3 +11,157 @@ Proxy / SSH Tunnel Connection Tab
+             the fields found under this connection option, see :ref:`<ssh-fields>`.
+
+         * - SSH with Identity File 
+           - Connects a deployment using a SSH Tunnel and uses an Identity File 
+             to secure the connection. For information on SSH connections and 
+             the fields found under this connection option, see :ref:`<ssh-fields>`.
+
+         * - Socks5
+           - Connects a deployment using a seperate proxy server. For more 
+             information on the fields found under this connection option, see 
+             :ref:`<socks5>`. 
+
+      .. _ssh-fields:
+ 
+      SSH Connection
+      ~~~~~~~~~~~~~~
+
+      The following fields are used when connecting a deployment using 
+      :guilabel:`SSH with Password` or :guilabel:`SSH with Identity File`:
+
+      .. list-table::
+         :header-rows: 1 
+         :widths: 50 50
+      
+         * - Field
+           - Description
+
+         * - SSH Hostname
+           - Enter the bastion (jumpbox) hostname. This is the unique identifier 
+             (Fully Qualified Domain Name, or FQDN) for the machine to be accessed. 
+
+         * - SSH Port
+           - Provide the port used for the SSH connection. This defaults to 22, 
+             the standard port for SSH.
+
+         * - SSH Username
+           - The username of the profile to log into on the remote system. This 
+             will be the user for which you want to establish the SSH connection.
+
+         * - SSH Password 
+           - (Optional) The password used to secure the SSH connection. This is 
+             required if you are not using an identity file.
+
+         * - SSH Identity File
+           - Select the file from which the identity (private key) for SSH public 
+             key authentication is read.
+
+             Unix or OS X: If using OpenSSH, identity files are found in the ~/.ssh 
+             directory. By default, the private key files have one of the following 
+             file names:
+
+             - ``id_dsa``
+             
+             - ``id_ecdsa``
+
+             - ``id_ed25519``
+ 
+             - ``id_rsa``
+
+             On Windows, the location of the identify files depends on your choice 
+             of SSH client, such as PuTTY. 
+
+         * - SSH Passphrase
+           - (Optional) If your private key is encrypted, provide the passphrase 
+             used to decrypt your private key (stored in the specified identity file). 
+             A passphrase provides an extra layer of security for an SSH connection.  
+
+      Behavior
+      ````````
+
+      - In |compass|, :guilabel:`SSH Tunnel Connections` allow users to connect 
+        to replica sets. 
+
+      - Users can connect to replica sets using SSH Tunnels and 
+        :ref:`TLS / SSL connections <tls-ssl-tab>`. 
+
+      .. _socks5:
+
+      Socks5
+      ~~~~~~~
+
+      The following fields are under the :guilabel:`Socks5` tab when connecting 
```

Comment:
"The following fields are in the :guilabel:`Socks5` tab, and are used to connect to a deployment"?

Reply:
Done.